lib/fog/aws/models/cloud_watch/alarm.rb

```diff
@@ -28,21 +28,20 @@ class Alarm < Fog::Model
         attribute :unit, :aliases => 'Unit'
 
         def initialize(attributes)
-          attributes['EvaluationPeriods'] ||= 1
-          attributes['Namespace']         ||= 'AWS/EC2'
+          self.namespace ||= "AWS/EC2"
+          self.evaluation_periods ||= 1
           super
         end
 
         def save
           requires :id
           requires :comparison_operator
-          requires :evaluation_periods
```

Comment:
Could we reinstate these also, just in case? Otherwise someone could set them to nil explicitly and run into weird errors. Thanks!